pipelines/github-forking-pipeline.yml

```
@@ -0,0 +1,25 @@
+    include:
+      - main
+      - master
+  tags:
+    include:
+      - '*'
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: mirror-git-repository-vsts-task@1
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)